sphinxcontrib/needs/builder.py

```diff
@@ -86,3 +87,10 @@ def build_needs_json(app, exception):
     needs_builder = NeedsBuilder(app)
     needs_builder.set_environment(app.env)
     needs_builder.finish()
+    
+    
+def delete_temp_dir(app, exception):
```

Comment:
`builder.py` is more used for the ``needs`` builder.
Maybe we put this small function directly inside ``needs.py`` or the ``urils.py`` file?